packages/backend/migrations/20250117112902_add_tenant_to_wallet_address.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema
+    .alterTable('walletAddresses', (table) => {
+      table.uuid('tenantId').references('tenants.id').index()
+    })
+    .then(() => {
+      return knex.raw(
+        `UPDATE "walletAddresses" SET "tenantId" = (SELECT id from "tenants" LIMIT 1)`
+      )
+    })
+    .then(() => {
+      return knex.schema.alterTable('walletAddresses', (table) => {
+        table.uuid('tenantId').notNullable().alter()
+      })
+    })
+}
```

Comment:
@mkurapov Should we edit this migration for the other resources? It just seems important that we ensure it defaults to the correct tenant and running them at once aids in that. Frankly I even wonder about adding this to the one that seeds the initial operator for that reason, or using an env var instead of the LIMIT 1. Either option reduces assumptions about the state of things when the migrations are run.

For development purposes I see why we'd end up with multiple migrations per resource (because thats how we've split the work) but for deployment purposes I see that more as one change.

Reply:
My assumption here is that as long as we have the "seed tenant operator" migration first, we should be ok. Then we get:
1. seed operator tenant
2. add tenantId to wallet address and backfill it
3. add tenantId to incoming payments 
4. ...

given that we run the migrations at startup in order, in the backfill migrations, we can only ever fetch the single operator tenant in these migrations. 

The only problem I see here is that we run migrations manually, run the  "seed tenant operator" and create a new tenant without running the other migrations - unlikely to happen. So we either, leave it alone (add a sort by oldest, or use the operator tenant id directly

Reply:
yeah Im not recommending any changes here. just something to think about.